examples/FadeExample.m

```diff
@@ -60,6 +60,10 @@ - (void)viewDidLoad {
 
 @implementation FadeTransition
 
+- (NSTimeInterval)transitionDurationWithContext:(nonnull id<MDMTransitionContext>)context {
+  return 5;
```

Comment:
Let's return a smaller number like 0.3 here so that the demo isn't excessively long.